v8/third_party/heap/mmtk/mmtk.cc

```diff
@@ -121,9 +121,8 @@ AllocationResult Heap::Allocate(size_t size_in_bytes, AllocationType type, Alloc
   bool large_object = size_in_bytes > kMaxRegularHeapObjectSize;
   size_t align_bytes = (type == AllocationType::kCode) ? kCodeAlignment : (align == kWordAligned) ? kSystemPointerSize : (align == kDoubleAligned) ? kDoubleSize : kSystemPointerSize;
   // Get MMTk space that the object should be allocated to.
-  int space = (type == AllocationType::kCode) ? 3 : (type == AllocationType::kReadOnly) ? 4 : (large_object) ? 2 : 0;
   Address result =
-      reinterpret_cast<Address>(alloc(tph_mutator_, size_in_bytes, align_bytes, 0, space));
+      reinterpret_cast<Address>(alloc(tph_mutator_, size_in_bytes, align_bytes, 0, 0));
```

Comment:
What does the `0` mean in the change?

Reply:
This is a temporary fix to allocate all objects into NoGC immortal space. Since we've enabled the base plan spaces for NoGC.

This code will be removed in the future. I'll do this in another PR to allocate objects into the correct spaces.